src/components/scs/ScSheet/ScSheet.js

```diff
@@ -172,6 +190,35 @@ const ScSheet = ({ sc, classes, intl }) => {
     setWorkQualityFields(values);
   };
 
+  const handleChangeSkills = (type, i, propKey, event) => {
+    if (type === 'skillsInTheField') {
+      const values = [...skillsInTheFieldFields];
+      values[i][propKey] = event.target.value;
+      setSkillsInTheFieldFields(values);
+    } else if (type === 'impactOnTeam') {
+      const values = [...impactOnTeamFields];
+      values[i][propKey] = event.target.value;
+      setImpactOnTeamFields(values);
+    } else if (type === 'serviceQuality') {
+      const values = [...serviceQualityFields];
+      values[i][propKey] = event.target.value;
+      setServiceQualityFields(values);
+    } else if (type === 'impactOnCompany') {
+      const values = [...impactOnCompanyFields];
+      values[i][propKey] = event.target.value;
+      setImpactOnCompanyFields(values);
+    }
+  };
+
+  const handleChangeWeightPercentage = (type, value) => {
+    if (type === 'performance') {
```

Comment:
Please validate value to be 0 < x < 100. 
Now you can write even negative numbers and second field gets above 100%